actionpack/lib/action_dispatch/middleware/exception_wrapper.rb

```diff
@@ -103,17 +103,13 @@ def backtrace
     end
```

Comment:
e.g. add a `cause_backtrace` as well, and update the backtrace cleaner to take a `backtrace` arg instead of only cleaning the original exception's trace.